develop/blocks/basics.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Creating Blocks
+description: A guide for creating simple blocks.
+authors:
+  - apple502j
+---
+
+# Creating Blocks
+One of the most common things to do in your mod is adding custom blocks.
+
+A block, at its simplest form, has no functionality and only one "block state". A block state is a combination of properties, or variations, for the same block - things like rotations, crop age, or redstone activation state. For example, chains have 6 block states; 3 for the `axis` property, times 2 for the `waterlogged` property. A block can have only finite number of pre-defined block states. A block that needs more storage and functionality must have an associated "block entity". A block entity can store infinite, customizable data and perform logics every tick.
```

Comment:
See my overall review comment, this is a prime example of a bulky paragraph, you should really split this up.

```suggestion
Blocks in their most basic form are static objects with a single appearance. To gain variations (like rotation, growth stage, or activation state), a block uses "block states". 

A block can only have a limited number of these predefined states. If a block needs more complex data storage or the ability to perform actions, it requires an associated "block entity", which offers customizable storage and the capacity to run logic each tick.
```